webapp/src/components/AtlasPage/Map/ParcelsMap/ParcelsMap.js

```diff
@@ -273,14 +285,20 @@ export default class ParcelsMap extends React.Component {
       districts
     )
 
+    const publication =
+      parcel && parcel.publication_tx_hash in publications
```

Comment:
Not sure if there's a clean way to do it, but this check ( `parcel.publication_tx_hash in publications`  ) appears on a few places like [here](https://github.com/decentraland/marketplace/pull/83/files#diff-203aa324a110b282e92b6aeeff854fc4R26). Maybe we can add a `utils` method? either `IsParcelInPublications()` or `getParcelPublication()`.

Whay do you think?